src/api/layers/layers-external.api.js

```diff
@@ -4,23 +4,47 @@ import WMTSCapabilitiesParser from '@/api/layers/WMTSCapabilitiesParser.class'
 import log from '@/utils/logging'
 
 /** Timeout for accessing external server in [ms] */
-const externalServerTimeout = 30000
+const EXTERNAL_SERVER_TIMEOUT = 30000
+
+/**
+ * Sets the WMS url parameters
+ *
+ * @param {URL} url Url to set
+ * @param {string} language Language to use
+ * @returns {URL} Url with wms parameter
+ */
+export function setWmsUrlParameters(url, language) {
```

Comment:
that is a name a bit too generic for my taste, maybe something like `setGetCapWmsUrlParameters` (I know it's long...)
Or at least explain in the JSDoc that this is made to request the getCapabilites endpoint, and not other endpoint (such as `getMap` for tiles, etc...)

Reply:
Done